_pkgdown.yml

```diff
@@ -0,0 +1,99 @@
+destination: reference
+
+url: https://pkgs.rstudio.com/bookdown/
```

Comment:
I noticed that in the three pkgdown PRs, sometimes this url ends with `/`, and sometimes it doesn't. Personally I prefer a trailing slash to avoid the ambiguity (i.e., indicate that it points to a subfolder, not a file).

Reply:
Good catch ! thanks.